### PR TITLE
AOM-33: Fix the different style of OpenMRS breadcrumbs in the Add-On Manager

### PR DESCRIPTION
**Pull request by wanjikum**
## JIRA TICKET NAME:

[AOM-33](https://issues.openmrs.org/browse/AOM-33)
Fix the different style of OpenMRS breadcrumbs in the Add-On Manager

### SUMMARY:

We are using some different style of breadcrumbs in the OpenMRS OWA. Usually, It should come between the header and the body part. But in the Add-Ons Manager OWA, It is located inside the body part. So It should be moved between the header and body part as usual.

### CODE REVIEW

**Comment by wanjikum** [comment]
Hello everyone, am still working on the ticket.

**Comment by wanjikum** [comment]
Am done, please review.

File: tests/homepage/homepage.test.js [hunk 1]
```diff
@@ -30,7 +29,4 @@ describe('<Homepage />', () => {
         expect(homePage.find("Link").getNodes()).to.have.length(2);
     });
 
-    it('should mount the BreadCrumbComponent in itself', () => {
-        expect(homePage.contains( <BreadCrumbComponent/> )).to.equal(true);
-    });
 });
```
Comment (jomadoye) [inline tests/homepage/homepage.test.js]: Slight EOL error here.

Comment (wanjikum) [reply tests/homepage/homepage.test.js]: 😃 Thanks.

**Comment by jomadoye** [comment]
Looks good to me, good job @wanjikum

**Comment by wanjikum** [comment]
Thank you @andela-jomadoye

**Comment by paulupendo** [comment]
Good job Milly. 🌮

**Comment by wanjikum** [comment]
Thanks @paulupendo

**Comment by dkayiwa** [comment]
Can you take a look at the merge conflicts on this?

**Comment by wanjikum** [comment]
sure

**Comment by wanjikum** [comment]
@dkayiwa Good morning 😄 I have fixed the merge conflicts.

**Comment by jomadoye** [comment]
Hi, @wanjikum, You may want to note that this PR has a lot of conflict with [AOM-37](https://issues.openmrs.org/browse/AOM-37)

**Comment by jomadoye** [comment]
Good job @wanjikum.

**Pull request merged**